net/rpmsg/rpmsg_sockif.c

```diff
@@ -287,11 +286,11 @@ static int rpmsg_socket_ept_cb(FAR struct rpmsg_endpoint *ept,
       nxmutex_lock(&conn->recvlock);
       conn->sendsize = head->size;
 
-      if (conn->psock)
-        {
-          conn->sconn.s_flags |= _SF_CONNECTED;
-          _SO_SETERRNO(conn->psock, OK);
-        }
+      conn->sconn.s_flags |= _SF_CONNECTED;
+#ifdef CONFIG_NET_SOCKOPTS
+      conn->sconn.s_error = OK;
+#endif
+      set_errno(OK);
```

Comment:
why call set_errno here

Reply:
Replace _SO_SETERRNO for equivalent, _SO_SETERRNO change the errno

Reply:
should we create a new patch which change the first parameter of _SO_SETERRNO  to conn?